README.md

```diff
@@ -1,6 +1,16 @@
 # EU eHealthNetwork Digitial COVID Certificate value sets
 
-This repository contains the value sets referenced by the EU Digital COVID Certificate (DCC) [JSON Schema](https://github.com/ehn-dcc-development/ehn-dcc-schema).
+This repository contains a snapshot of the value sets referenced by the EU Digital COVID Certificate (DCC) [JSON Schema](https://github.com/ehn-dcc-development/ehn-dcc-schema). The most up-to-date version of the valuesets are distributed on the DCCG (DCC Gateway).
+
+
+Name | Description
+-----| ------------------
+test-manf | List of recognised manufactorers of tests. Snapshot provided for development purposes, please refer to the gateway for the actual version.
+test-result | List of supported test results (i.e. detected, not detected..)
+test-type | List of supported test types (i.e. PCR, rapid-antigen..)
+disease-agent-targeted | List of diseases/targeted agent 
+country-2-codes | List of ISO-3166-2 country codes
+
 
 Release: 2.9.0
```

Comment:
For completeness' sake: the addition of this information should be documented in `CHANGELOG.md` as well.

Reply:
I would support taking the version out of the readme. We have this information already in the "Releases" and the `CHANGELOG.md` 

We could put here links to these two locations